src/components/Lesson/Lesson.js

```diff
@@ -28,10 +31,15 @@ const Lesson = React.createClass({
       this.props.lesson.content = renderScratchBlocks(content);
     }
   },
+  getLevelIcon(level) {
+    return level != null
+      ? <img className={styles.levelIcon} src={require('../../assets/graphics/level-' + level + '.svg')}/>
+      :  null;
+  },
   render() {
     return (
       <div>
-        <h1>{this.getTitle()} - Level {this.getLevel()}</h1>
+        <h1>{this.getLevelIcon(this.getLevel())}{this.getTitle()} - Level {this.getLevel()}</h1>
```

Comment:
Er det ikke lettere om getLevelIcon() bare kaller getLevel() på egenhånd? Det er vel ikke noe behov for at level blir sendt med inn som parameter?